innersource-guidance-group.md

```diff
@@ -0,0 +1,43 @@
+## Title
+InnerSource Guidance Group
+
+## Patlet
+A company may require a mechanism to set broad governance and behavioral guidelines as part of the rollout and sustaining needs of an InnerSource initiative.
+
+## Problem
+Without a set of organizational standards Trusted Committers, and individual teams, are free to set their own standards, which is a step in the right direction.  However, team standards could be highly variant and further diverge over time as they evolve and adapt to new needs and forces.  This can lead to confusion and frustration as contributors submit code across team and functional boundaries. The business is interested establishing a set of agreed-upon standard operating procedures that guide teams while still providing sufficient autonomy to meet the parochial needs of individual teams.
+
+## Context
+A company is planning to introduce InnerSource as a strategic initiative. They are faced with reforming and steering the behavior of developers and engineers toward a common goal.  In general functional groups and teams within the company adhere to a broad spectrum of standards regarding tools, development languages, collaboration models, engagement models, documentation models, etc. The company recognizes the value of the Trusted Committer role as a situational leader for a team that sets and enforces team standards, code contribution standards, contribution agreements, PR management, etc. however the company is interested in establishing organizational guidelines as well that steer and focus Trusted Committers and the teams they represent.
+
+As an example, the breadth of languages used within an engineering and operations organization could include, powershell, bash shell, perl, tcl, python, golang, java, shell augmented with sed and awk and jq, etc. and the organization wants to standardize as much as possible on a subset of these languages so that the ability to collaborate between teams is maximized.
+
+## Forces
+- Separate engineering and development teams establish development standards that may be highly divergent from other teams which limits the ability to collaborate.
+- As people move between teams, or responsibility for specific initiatives change over time, the ability of the organization to adapt to variant team norms results in decreased development velocity.
+
+## Solutions
+Recognize that Trusted Committers are both situational leaders and subject matter experts representing their team and establish a governance model with a “committee” attended and chaired by Trusted Committers chartered with establishing company and/or organizational guidelines.
```

Comment:
I think that `governance model` is a loaded term, and maybe too strong for what you actually describe this group as doing (establish organization guidelines).

Reply:
I agree, governance might be too strong.

I have reworded this like this:

> Recognize that Trusted Committers are both **situational leaders** and **subject matter experts** representing their team. Establish an InnerSource Guidance Group as a “committee” attended and chaired by Trusted Committers. Charter this InnerSource Guidance Group with establishing company and/or organizational guidelines.